src/IIS/Settings/Bindings/IISBindings.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using Cake.IIS.Settings.Bindings.FluentAPI;
+
+namespace Cake.IIS.Settings.Bindings
+{
+    internal class IISBindings : IBindings
```

Comment:
This should be a static class, exposing the implementations from static functions